src/deploy/ecs.py

```diff
@@ -47,13 +47,21 @@ def list_service_arns(self, *, cluster_arn):
         for page in paginator.paginate(cluster=cluster_arn):
             yield from page["serviceArns"]
 
-    def redeploy_service(self, cluster_arn, service_arn):
+    def redeploy_service(self, cluster_arn, service_arn, deployment_label):
         response = self.ecs.update_service(
             cluster=cluster_arn,
             service=service_arn,
             forceNewDeployment=True
         )
 
+        self.ecs.tag_resource(
+            resourceArn=service_arn,
+            tags=[{
+                'key': 'deployment:label',
+                'value': deployment_label
+            }]
+        )
```

Comment:
This will blat any other tags on the resource. It's a bit more work, but I think we should follow the same good habits in the storage service – read the existing tags, update them, write them back.

Reply:
We must not blat the tags, as they are required for weco-deploy elsewhere ... though I am not certain boto3 behaves the same way. I have actually tested this without removing the other tags so I think boto3 behaves differently. I'll double check.

Reply:
https://boto3.amazonaws.com/v1/documentation/api/1.9.88/reference/services/ecs.html#ECS.Client.tag_resource
>  If existing tags on a resource are not specified in the request parameters, they are not changed.

Reply:
That is absolutely not what I expected (and apparently I misread the docs), but yay!

Reply:
It's what the SDK does, but not boto3 - so yes confusing.